app/src/main/res/layout/item_product_order.xml

```diff
@@ -67,44 +72,82 @@
+                            android:layout_marginTop="@dimen/dp_2"
+                            android:gravity="center_vertical"
+                            android:text="@{viewModel.quantity}"
+                            />
+                        <TextView
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_marginTop="@dimen/dp_2"
+                            android:text="@{viewModel.price}"
+                            />
+                    </LinearLayout>
+                    <LinearLayout
+                        android:id="@+id/layout_button"
+                        android:layout_width="@dimen/dp_35"
+                        android:layout_height="wrap_content"
                         android:layout_alignParentEnd="true"
                         android:layout_alignParentRight="true"
+                        android:gravity="center_horizontal"
+                        android:orientation="vertical"
+                        >
+                        <ImageButton
+                            android:layout_width="@dimen/dp_30"
+                            android:layout_height="@dimen/dp_30"
+                            android:background="@drawable/ic_accept"
+                            android:onClick="@{() -> viewModel.onClickAcceptProduct()}"
+                            tools:ignore="ContentDescription"
+                            />
+                        <ImageButton
+                            android:layout_width="@dimen/dp_30"
+                            android:layout_height="@dimen/dp_30"
+                            android:layout_marginTop="@dimen/dp_10"
+                            android:background="@drawable/ic_reject"
+                            android:onClick="@{() -> viewModel.onClickRejectProduct()}"
+                            tools:ignore="ContentDescription"
+                            />
+                    </LinearLayout>
+                    <TextView
+                        android:id="@+id/text_note"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_below="@+id/layout_top"
+                        android:text="@string/note"
+                        />
+                    <TextView
+                        android:layout_width="match_parent"
+                        android:layout_below="@+id/layout_top"
+                        android:layout_toEndOf="@+id/text_note"
+                        android:layout_toLeftOf="@+id/layout_button"
+                        android:layout_toRightOf="@+id/text_note"
+                        android:layout_toStartOf="@+id/layout_button"
+                        android:paddingLeft="@dimen/dp_5"
+                        android:layout_height="wrap_content"
+                        android:paddingStart="@dimen/dp_5"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingStart` you should probably also define `paddingEnd` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.